playground_convert.py

```diff
@@ -0,0 +1,38 @@
+from pathlib import Path
```

Comment:
Why put this playground file in the root directory

Reply:
Its inappropriate.

Reply:
The main purpose of this pull request is to show my process of debugging. The playground file is a kind of test file, but it does not run successfully, so I don't put it in test directory. Now I've deleted the playground file and it's appropriate for merge now.